addons/website/static/tests/tours/edit_translated_page.js

```diff
@@ -0,0 +1,19 @@
+/** @odoo-module **/
+
+import tour from "web_tour.tour";
+
+tour.register('edit_translated_page_redirect', {
+    test: true,
+    url: '/nl/contactus',
```

Comment:
as discussed, merge both tests in the same test (naviguate to contactus then to homepage) so we dont spawn 2 process/browser/routing map/cold state etc
Then should be good to r+